### DISCUSSION TITLE
Embedding movies to a website

### DISCUSSION THREAD

**Discussion by GoldieBowyien** [GoldieBowyien/Movie_streaming#10] [Q&A] [answered]
How do you embed movies into a website?

**Comment by NeoDev99** [comment, marked as answer]
To Embed videos & movies on a website you must:
1) On a computer, go to the YouTube video or playlist you want to embed.
2) Click SHARE .
3) From the list of Share options, click Embed.
4) From the box that appears, copy the HTML code.
5) Paste the code into your website HTML.

**Reply by GoldieBowyien** [reply]
Thanks your answer was helpful
